### PR TITLE
Fix failing test by testing clearinghouse

### PR DESCRIPTION
**Pull request by daaang**

### CODE REVIEW

File: manifests/profile/clearinghouse/s3backup.pp [hunk 1]
```diff
@@ -10,10 +10,10 @@
 #   include nebula::profile::clearinghouse::s3backup
 
 class nebula::profile::clearinghouse::s3backup (
+  String $bucket,
   String $mail_recipient = lookup('nebula::automation_email'),
   String $clearinghouse_filesystem = '/clearinghouse',
   String $backup_path = 'clearinghouse',
-  String $bucket
```
Comment (daaang) [inline manifests/profile/clearinghouse/s3backup.pp:16]: This was to appease the linter.

File: manifests/profile/clearinghouse/s3backup.pp [hunk 2]
```diff
@@ -25,7 +25,7 @@
 
     'upload /clearinghouse to s3':
       command => "/usr/bin/aws s3 sync --delete --quiet ${clearinghouse_filesystem} ${bucket}/${backup_path}",
-      day     => 7,
+      weekday => 7,
```
Comment (daaang) [inline manifests/profile/clearinghouse/s3backup.pp:28]: An error uncovered once there was a test

File: spec/fixtures/hiera/clearinghouse_config.yaml [hunk 3]
```diff
@@ -2,7 +2,7 @@
 :backends:
   - yaml
 :hierarchy:
-  - chipmunk
+  - clearinghouse
```
Comment (daaang) [inline spec/fixtures/hiera/clearinghouse_config.yaml:5]: Not sure why the clearinghouse hiera config was pointing at chipmunk, but fixed that.

File: spec/fixtures/hiera/clearinghouse.yaml [hunk 4]
```diff
@@ -1 +1,3 @@
 nebula::profile::mysql::password: changeme
+nebula::profile::ssl_keypair::chain_crt: intermediate_ca.crt
```
Comment (daaang) [inline spec/fixtures/hiera/clearinghouse.yaml:2]: Copied this from the chipmunk hiera; tests wouldn't pass without it.

File: spec/classes/profile/clearinghouse/s3backup_spec.rb [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+# BSD License. See LICENSE.txt for details.
+require 'spec_helper'
+
+describe 'nebula::profile::clearinghouse::s3backup' do
+  on_supported_os.each do |os, os_facts|
+    context "on #{os}" do
+      let(:hiera_config) { 'spec/fixtures/hiera/clearinghouse_config.yaml' }
+      let(:facts) { os_facts }
+
+      it { is_expected.to compile }
```
Comment (daaang) [inline spec/classes/profile/clearinghouse/s3backup_spec.rb:14]: All this does is check that the s3backup profile compiles.

**Comment by rrotter** [comment]
Ouch. Thanks for nailing down the copy-pasta.

**Pull request merged**